PWGMM/Tasks/run2dndeta.cxx

```diff
@@ -205,7 +251,7 @@ struct PseudorapidityDensity {
     }
   }
 
-  PROCESS_SWITCH(PseudorapidityDensity, processGen, "Process generator-level info", false);
```

Comment:
Due to limitations of process switches it is better to have them `false` by default, since the configuration file with have all of them set anyway.